autoload/airline/extensions/denite.vim

```diff
@@ -14,7 +14,17 @@ function! airline#extensions#denite#check_denite_mode(bufnr)
   if &filetype != 'denite'
     return ''
   endif
-  let mode = split(denite#get_status_mode(), ' ')
+  if exists('*get_status_mode')
```

Comment:
Does this make sense, really? Should it be `exists('*denite#get_status_mode')` ?

Reply:
you are right, will fix it